stages/01_process.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import numpy as np
+import pandas as pd
+from pathlib import Path
+import pyarrow.lib as pa
+import zipfile
+
+logging.basicConfig(format="%(asctime)s %(message)s")
+logger = logging.getLogger(__name__)
+logger.setLevel("INFO")
+
+
+def create_unzip_dir():
+    logger.info("creating unzip/ directory")
+    unzip_dir = Path("unzip")
+    unzip_dir.mkdir(exist_ok=True)
+    return unzip_dir
+
+
+def unzip_file(f):
+    file_path = Path(f)
+    logger.info("unzipping file at %s", file_path)
+    with open(file_path, "rb") as zip:
+        try:
+            zip_root = zipfile.ZipFile(zip)
+            zip_root.extractall(Path("unzip"))
```

Comment:
Can we unzip to a directory with the same base name as the `.zip` file? That way the files will be grouped together in directories under the `brick/` directory as well.